ci/github-script/merge.js

```diff
@@ -295,6 +272,18 @@ async function handleMerge({
       '',
     ]
 
+    if (eligible.size > 0 && !eligible.has(comment.user.id)) {
+      const users = await Promise.all(
+        Array.from(eligible).map(async (id) => (await getUser(id)).login),
```

Comment:
We can do this (I think):

```suggestion
        Array.from(eligible, async (id) => (await getUser(id)).login),
```